<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProtoQt Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/images/redmine_icon.png</normaloff>:/images/images/redmine_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>177</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>false</bool>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Connection</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/images/redmine_icon.png</normaloff>:/images/images/redmine_icon.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Redmine defaults</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/images/redmine_icon.png</normaloff>:/images/images/redmine_icon.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Notifications</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/images/help-hint.png</normaloff>:/images/images/help-hint.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="pageConnection">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Redmine connection</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>URL</string>
              </property>
              <property name="buddy">
               <cstring>redmineUrlEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="redmineUrlEdit">
              <property name="placeholderText">
               <string>https://prj.adyax.com</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="basicAuthCheckBox">
              <property name="text">
               <string>Basic auth</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Login</string>
              </property>
              <property name="buddy">
               <cstring>basicAuthLoginEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="basicAuthLoginEdit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Password</string>
              </property>
              <property name="buddy">
               <cstring>basicAuthPassEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="basicAuthPassEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>API access key</string>
              </property>
              <property name="buddy">
               <cstring>redmineApiKeyEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="redmineApiKeyEdit">
              <property name="placeholderText">
               <string>cbcdbe476f5dcb4ba8a3e02cd2b4ff718efef407</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageRedmineDefs">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Activity to use</string>
           </property>
           <property name="buddy">
            <cstring>comboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Fetch issues with statuses</string>
           </property>
           <property name="buddy">
            <cstring>listWidget_2</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QListWidget" name="listWidget_2">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Fetch issues where I'm Watcher</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>redmineUrlEdit</tabstop>
  <tabstop>redmineApiKeyEdit</tabstop>
  <tabstop>basicAuthCheckBox</tabstop>
  <tabstop>basicAuthLoginEdit</tabstop>
  <tabstop>basicAuthPassEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
